addon-test-support/audit.ts

```diff
@@ -111,10 +90,10 @@ export function _normalizeRunParams(
   }
 
   if (typeof options !== 'object') {
-    options = config[_configName]?.componentOptions?.axeOptions || {};
+    options = getRunOptions() || {};
```

Comment:
Why do we need to default? I think `axe-core` can handle `undefined` for options...

Reply:
This was the old behavior, so this was just preserving it.

Reply:
I think I can probably remove it, since that param is optional anyway.